project/profiles/capacity4more/modules/c4m/user/c4m_user_og/c4m_user_og.module

```diff
@@ -915,3 +944,33 @@ function c4m_user_og_user_update(&$edit, $account, $category) {
   // Destroy the entry of this account.
   c4m_user_og_delete_unified_workflow_entries($account->uid);
 }
+
+/**
+ * Return true if the user is allowed to invite and false if he isn't.
+ *
+ * Checks by user role and og_role against group visibility (Public/Private) and
+ * Membership requests setting (Open/Moderated).
+ */
+function c4m_user_og_invite_access($gid, $group_type = 'node') {
+  global $user;
+  $og_roles = og_get_user_roles($group_type, $gid);
+
+  // Global and Group administrators can invite.
+  if (in_array('administrator', $user->roles) || in_array('administrator member', $og_roles)) {
```

Comment:
please use `c4m_user_og_get_role_in_group()` and `user_access('administer site') in order to check SA/GO/GA